core/src/test/scala/unit/kafka/log/LogCleanerManagerTest.scala

```diff
@@ -181,6 +181,7 @@ class LogCleanerManagerTest extends JUnitSuite with Logging {
 
     val config = LogConfig(logProps)
     val partitionDir = new File(logDir, "log-0")
+    partitionDir.mkdir()
```

Comment:
Hmm. This is covering a chicken/egg situation around the initialisation of Log (i.e. where we initialise leaderEpochCache). Needs changing.